# Kp: [650.0, 650.0, 650.0, 650.0, 100.0, 100.0, 50.0]
# Kp: [350.0, 350.0, 350.0, 350.0, 100.0, 100.0, 50.0]
#    "joint_stiffness: [200.0, 200.0, 150.0, 150.0,
#                       100.0, 90.0, 50.0]"
# Kp: [200.0, 200.0, 150.0, 150.0, 100.0, 90.0, 50.0]
# Kd: [30.0, 30.0, 30.0, 30.0, 5.0, 5.0, 2.0]
Kp: [100.0, 100.0, 100.0, 100.0, 75.0, 40.0, 30.0]
# Kp: [150.0, 150.0, 150.0, 100.0, 75.0, 40.0, 30.0]
Kd: [20.0, 20.0, 20.0, 20.0, 7.5, 5.0, 2.0]
Km: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
Ki: [175.0, 150.0, 150.0, 250.0, 120.0, 70.0, 50.0]

i_thresh: [0.025, 0.025, 0.025, 0.025, 0.025, 0.025, 0.025]
i_min: [0.00001, 0.00001, 0.00001, 0.00001, 0.0001, 0.0001, 0.0001]
saturate_tau: [1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0]
# min_tau: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
# offset_tau: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
alpha_q: 0.7
alpha_dq: 0.1
